machine/fsl_t2080rdb/test/fsl_p2020rdbpca_dut.py

```diff
@@ -0,0 +1,33 @@
+# FSL T2080RDB DUT Class
```

Comment:
No need for this file.  The testing architecture has changed.  See https://github.com/opencomputeproject/onie/wiki/Testing-and-Certification